certify-default.properties

```diff
@@ -2,6 +2,18 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+### DataProvider plugin specific changes
+# tells Certify to become a proxy to other issuers via VCIssuancePlugin impl
+mosip.certify.issuer=PluginIssuer
```

Comment:
Comment needs to be improved with the supported options and clear explanation what option means what.

Reply:
this level comment to be provided for every new property that is added in all the property files

Reply:
Updated comment with all supported values for this property and also added specific comments for new properties added.